src/main/drivers/pwm_output_dshot_hal.c

```diff
@@ -94,6 +94,51 @@ void pwmWriteDigital(uint8_t index, uint16_t value)
     }
 }
 
+void pwmWriteProShot(uint8_t index, uint16_t value)
+{
+    if (!pwmMotorsEnabled) {
```

Comment:
Same here, it becomes easier to maintain if the code is reused, instead of copy / pasted.